lib/Dialect/AIEX/Transforms/AIECreateLocks.cpp

```diff
@@ -160,8 +160,8 @@ struct AIECreateLocksPass : public AIECreateLocksBase<AIECreateLocksPass> {
       Operation *acqUser = TA.getTokenUserOp(acquire);
       bool IsRelUserCore = isa<CoreOp>(relUser);
       bool IsAcqUserCore = isa<CoreOp>(acqUser);
-      std::pair<int, int> relUserCoord = TA.getCoord(relUser);
-      std::pair<int, int> acqUserCoord = TA.getCoord(acqUser);
+      [[maybe_unused]] std::pair<int, int> relUserCoord = TA.getCoord(relUser);
+      [[maybe_unused]] std::pair<int, int> acqUserCoord = TA.getCoord(acqUser);
```

Comment:
Have you tried moving these within the LLVM_DEBUG block?

Reply:
I have not, I will give it a try!